.github/workflows/release.yml

```diff
@@ -79,8 +79,74 @@ jobs:
       with:
         name: encore-${{ github.event.inputs.version }}-${{ matrix.goos }}_${{ matrix.goarch }}
         path: encore-${{ github.event.inputs.version }}-${{ matrix.goos }}_${{ matrix.goarch }}.tar.gz
-  notify_release_success:
+
+  publish-docker-images:
+    name: "publish docker images"
+    runs-on: ubuntu-20.04
     needs: build
+    permissions:
+      contents: read
+      packages: write
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          sparse-checkout: .github
+      - name: Download Artifacts
+        uses: actions/download-artifact@v3
+        with:
+          path: .github/dockerimg/artifacts
+      - name: Setup Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Login to Docker Registry
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      - name: Cache Docker layers
+        uses: actions/cache@v2
+        with:
+          path: /tmp/.buildx-cache
+          key: ${{ runner.os }}-buildx-${{ github.sha }}
+          restore-keys: |
+            ${{ runner.os }}-buildx-
+
+      - name: Extract metadata (tags, labels) for Docker
+        id: meta
+        uses: docker/metadata-action@v3
+        with:
+          images: encoredotdev/encore
```

Comment:
damn was `encoredev` not free?

Reply:
Nope, unfortunately not, but this matches our Twitter handle so it's not without precedent